web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/traversing/var/siblings.js

```diff
@@ -0,0 +1,17 @@
+define( function() {
```

Comment:
:no_entry_sign: **Error**: Opening parenthesis of a multi-line function call must be the last content on the line (*PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket*).